app/controllers/optOut/ConfirmOptOutController.scala

```diff
@@ -22,38 +22,42 @@ import config.featureswitch.FeatureSwitching
 import config.{AgentItvcErrorHandler, FrontendAppConfig, ItvcErrorHandler}
 import connectors.optout.OptOutUpdateRequestModel.OptOutUpdateResponseSuccess
 import controllers.agent.predicates.ClientConfirmedController
-import models.optout.OptOutCheckpointViewModel
+import models.optout.{MultiYearOptOutCheckpointViewModel, OneYearOptOutCheckpointViewModel, OptOutCheckpointViewModel}
```

Comment:
repeating code, can be pulled in a rusable method
Logger("application").error("No qualified tax year available for opt out")
errorHandler(isAgent).showInternalServerError()

Reply:
or add a todo comment about this

Reply:
This is the only use of it does it require a reuseable method?